content/python/concepts/strings/strings.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Strings'
-Description: 'A string is a sequence of characters contained within a pair of single quotes or double quotes. Strings can be of any length and can contain letters, words, spaces, and symbols. Other data types such as integers, doubles, and booleans can also be strings so long as they are wrapped in quotes.'
+Description: 'A string is a sequence of characters contained within a pair of single quotes or double quotes.'
```

Comment:
Can we change to "single or double quotes" rather than "single quotes or double quotes"